src/styles/settings/_z-indices.scss

```diff
@@ -0,0 +1,10 @@
+// intervals of 100 leave plenty of room for exceptions.
+$z-indices: (
+  "z1": 100,
+  "z2": 200,
+  "z3": 300,
+  "z4": 400,
+  "z5": 500,
+  "z6": 600,
+  "z7": 700,
+);
```

Comment:
Awesome idea here! We haven't used z-index a ton, but this should come in handy as the site grows.